packages/event-emitter/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@hono/event-emitter",
+  "version": "1.0.0",
```

Comment:
The `version` field will be updated automatically by the Changesets. So, in this case, you should write it `0.0.0` instead of `1.0.0`.